scripts/sct_process_segmentation.py

```diff
@@ -589,13 +589,32 @@ def extract_centerline(fname_segmentation, remove_temp_files, verbose = 0, algo_
         file_results.write(str(int(i)) + ' ' + str(x_centerline_voxel[i - min_z_index]) + ' ' + str(y_centerline_voxel[i - min_z_index]) + '\n')
     file_results.close()
 
+    # create a .roi file
+    from datetime import datetime
+    date_now = datetime.now()
+    common_text_start = 'Begin Marker ROI\n  Build version="7.0_33"\n  Annotation=""\n  Colour=0\n  Image source="' + fname_segmentation + '"\n  Created  "' + date_now.strftime("%d %B %Y %H:%M:%S.%f %Z") + '" by Operator ID="SCT"\n'
```

Comment:
The string is long could be hard to debug. I would suggest creating a template like the one below and use string format and populate it by argument name such as shown here https://docs.python.org/2/library/string.html#format-examples
```
ROI_TEMPLATE="""Begin Marker ROI
Build version="7.0_33"
Annotation=""
Colour=0
  Image source="{fname_segmentation}"
  Created  "{creation_date}" by Operator ID="SCT"
  Slice={slice_num}
  Begin Shape
    X={shape_x}; Y={shape_y};
  End Shape
End Marker ROI
"""
```